pages.zh/common/df.md

```diff
@@ -0,0 +1,24 @@
+# df
+
+> 提供文件系统磁盘空间使用情况的概览。
+> 更多信息：<https://www.gnu.org/software/coreutils/df>.
+
+- 显示所有文件系统和它们的磁盘使用情况：
+
+`df`
+
+- 以人类可读的形式显示所有文件系统和它们的磁盘使用情况：
+
+`df -h`
+
+- 显示包含给定文件或目录的文件系统及其磁盘使用情况:
+
+`df {{文件或目录的路径}}`
```

Comment:
```suggestion
`df {{路径/到/文件或目录}}`
```
Reference: <https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md>.

Reply:
Thank you for your suggestion. @kbdharun